docs/contributing/docs-templates.md

```diff
@@ -7,487 +7,13 @@ Docs reference guides, recipes, and tutorials teach Gatsby concepts to users wit
-
-### Dependencies (optional)
-
-Are there any plugins that must be installed in order to make this plugin work? If so, please include a list of those plugins and links to their pages here.
-
-### Learning Resources (optional)
-
-If there are other tutorials, docs, and learning resources that are necessary or helpful to someone using this plugin, please link to those here.
-
-## How to install
-
-Please include installation instructions here.
-
-Gatsby documentation uses `npm` for installation. This is the recommended approach for plugins as well.
-
-If the plugin is a theme that needs to use `yarn`, please point to [the documentation for switching package managers](/docs/gatsby-cli/#how-to-change-your-default-package-manager-for-your-next-project) in addition to the `yarn`-based instructions.
-
-## Available options (if any)
-
-## When do I use this plugin?
-
-Include stories about when this plugin is helpful and/or necessary.
-
-## Examples of usage
-
-This usually shows a code example showing how to include this plugin in a site's `config.js` file.
-
-    code example
-
-//See this [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) on how to format code examples.
-
-This section could also include before-and-after examples of data when the plugin is enabled, if applicable.
-
-## How to query for data (source plugins only)
-
-If this is a source plugin README, source plugins ought to allow people to query for data within their Gatsby site. Please include code examples to show how to query for data using your source plugin.
-
-If this is a theme that requires data in a specific format in order to match an existing query, include those examples here.
-
-## How to run tests
-
-## How to develop locally
-
-## How to contribute
-
-If you have unanswered questions, would like help with enhancing or debugging the plugin, it is nice to include instructions for people who want to contribute to your plugin.
-```
-
----
-
-## Starter README template
-
-### Near-perfect example of a starter README
-
-[Default Starter README](https://github.com/gatsbyjs/gatsby-starter-default)
-
-```markdown
-## Name of starter
-
-## Quick start
-
-Give instructions on how to install this starter
-
-## Features
-
-Tell features comes with this starter. This is a chance to give users a brief tour of how to use this starter effectively.
-
-## Next steps
-
-Any tips on how to deploy this starter? What CMS to use? Other fun ways to build on top of the starter? Say those here!
-```
+Follow the how-to templates (models) listed here when contributing to Gatsby docs to ensure that the docs accomplish their purpose. If you have a good reason to deviate from the following template structures, mention those reasons in the PR so others can give proper feedback.
```

Comment:
```suggestion
Follow the how-to templates (models) listed here when contributing to Gatsby docs to ensure that the docs accomplish their purpose. If you have a good reason to deviate from the following template structures, mention those reasons in an issue so others can give proper feedback on your idea.
```

We need to bias toward issues before PRs for discussion with @gatsbyjs/learning, especially if someone wants to deviate from expected formats.

With subpages under this overview page in `contributing-links`, you could also ideally use the `<Guidelist />` component to automatically list them: https://www.gatsbyjs.org/contributing/docs-and-blog-components/#guide-list

Reply:
@marcysutton thank you so much for taking the time to go over these documents and providing feedback on each change and its necessity. I do not have the level and depth of each particular template/doc so this was a great learning experience for me. 😃